spec/schemas/snapshot._common.yaml

```diff
@@ -11,159 +11,188 @@ components:
       properties:
         data_streams:
           type: array
+          description: The list of data streams included in the snapshot.
           items:
             type: string
         duration:
           $ref: '_common.yaml#/components/schemas/Duration'
+          description: The total time taken to complete the snapshot.
         duration_in_millis:
           $ref: '_common.yaml#/components/schemas/DurationValueUnitMillis'
+          description: The total time taken to complete the snapshot in milliseconds.
         end_time:
           $ref: '_common.yaml#/components/schemas/DateTime'
+          description: The time when the snapshot completed.
         end_time_in_millis:
           $ref: '_common.yaml#/components/schemas/EpochTimeUnitMillis'
+          description: The time when the snapshot completed in milliseconds.
         failures:
           type: array
+          description: The list of shard failures that occurred during the snapshot.
           items:
             $ref: '#/components/schemas/SnapshotShardFailure'
         include_global_state:
           type: boolean
+          description: Whether the snapshot includes the cluster state.
         indices:
           type: array
+          description: The list of indices included in the snapshot.
```

Comment:
Vale linter complains about the use of `indices.

Reply:
Fixed.